test/evergreen.yml

```diff
@@ -2067,6 +2067,39 @@ tasks:
             cp ../../../bench/wtperf/split_heavy.wtperf .
             ./wtperf -O ./split_heavy.wtperf -o verbose=2
 
+  - name: static-wt-build-test
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          posix_configure_flags: --disable-shared --with-builtins=snappy,zlib
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          shell: bash
+          script: |
+            set -o errexit
+            set -o verbose
+
+            # Delete wt util
+            rm -f wt
+            # Build static wt util
+            make CC='eval "g++ -static"'
```

Comment:
Can the make part being covered by the above `compile wiredtiger` function?

Reply:
The idea here is to
1. Build wiredtiger with "--disable-shared --with-builtins=snappy,zlib" configure option and default compiler(CC=mongodbtoolchain-gcc, CXX=mongodbtoolchain-g++). 
2. Delete wt util
3. Rebuild/remake wiredtiger with -static option.
4. Check if "wt" util is statically linked.

Reply:
Thanks for the clarification. Now it makes sense to me. 
Would it be cleaner to call `make wiredtiger` function here, or at least adding the `${smp_command}` in the make command?

Reply:
Thanks for pointing it out. I cannot call "make wiredtiger" function here because CC variable cannot be modified under this function to be set to "g++ -static". 
Added "${smp_command}" instead in the make command.

Reply:
Sounds good, thanks.